src/javascript/JContent/actions/deleteActions/Delete/Delete.jsx

```diff
@@ -86,7 +86,7 @@ const DeleteContent = ({data, onClose, isLoading, isMutationLoading, dialogType,
                     <DialogContentText className={styles.content} dangerouslySetInnerHTML={{__html: label}}/>
                     {hasUsages && count === 1 &&
                         <DialogContentText>{t('jcontent:label.contentManager.deleteAction.hasUsages.single')}</DialogContentText>}
-                    {hasUsages && count > 1 &&
+                    {hasUsages && (count > 1 || count === 0) &&
```

Comment:
trying to understand how we can have usages but with 0 count. is this because of permissions? 

`count` conditional also doesn't look like it's necessary here and can be removed, but also needs to be replaced by a check if theres usagesOverflow (otherwise case below won't trigger)

Reply:
ok understood with the permissions.

The count conditional is only checking if it's a non-negative number which will always be the case (the singular case is already taken care of above), base on the count calculation. Even in the case where count ends up with an invalid value (e.g. count=undefined or NaN) maybe it's safer to display this as a fail-safe message as long as hasUsages is true.

also ignore the usagesOverflow comment, I misread read the conditional as `hasUsages && usagesOverflow`

Reply:
after discussion, we've simplified the conditional but still need it here as we can end up with two messages when count == 1